src/lib.rs

```diff
@@ -44,6 +45,7 @@ pub async fn serve(config: &Config) -> Result<()> {
         .route("/domains", get(list_domains_handler))
         .route("/lnaddress/:domain/:username", get(get_lnaddr_handler))
         .route("/lnaddress/register", post(register_lnaddr_handler))
+        .route("/lnaddress/remove", post(remove_lnaddr_handler))
```

Comment:
Nit: Should this be a `DELETE` request instead? The register one has to be post since it's not idempotent (as we discussed today due to the auth token being returned).

Reply:
Yes, will fix